<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>layerselector</class>
 <widget class="QWidget" name="layerselector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>195</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListView" name="listView">
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="del_button">
     <property name="toolTip">
      <string>Delete selected layers</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="add_button">
     <property name="toolTip">
      <string>Add copies of selected layers at the end, or a new layer if nothing is selected</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="select_label">
     <property name="toolTip">
      <string>Python style range selector</string>
     </property>
     <property name="text">
      <string>Select:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="select_lineEdit">
     <property name="toolTip">
      <string>Python style range selector</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>add_button</sender>
   <signal>clicked()</signal>
   <receiver>layerselector</receiver>
   <slot>addClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>593</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>del_button</sender>
   <signal>clicked()</signal>
   <receiver>layerselector</receiver>
   <slot>delClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>select_lineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>layerselector</receiver>
   <slot>selectionEntered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>662</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>delClicked()</slot>
  <slot>addClicked()</slot>
  <slot>selectionEntered()</slot>
  <slot>selectionChanged()</slot>
 </slots>
</ui>
